app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -99,7 +99,16 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
                 class_name = "middleware_deployment";
                 break;
             case "MiddlewareServer":
-                class_name = "middleware_server";
+                switch (d.item.product) {
+                    case "Hawkular":
+                        class_name = "vendor-hawkular";
```

Comment:
Hawkular as MW-provider can monitor MW-Servers of various kinds and especially does so for the Hawkular-Server itself. The different types/kinds of servers are also reflected in the Product and the icon in the list view and the quadicon inside a single listed server.

![bildschirmfoto 2016-04-06 um 10 49 10](https://cloud.githubusercontent.com/assets/208246/14310981/47c4a526-fbe5-11e5-9282-0d9fb90e4efc.png)